src/Actions/CodeUpAction.php

```diff
@@ -115,9 +115,9 @@ public function run(?string $stage = null)
         }
 
         try {
-            $this->section("git push ($msg)");
+            $this->section("git push ($msg) to $upstream from $branch to master");
             $git->getWorkingCopy()->getWrapper()->streamOutput();
-            $git->push($upstream, 'master');
+            $git->push($upstream, "${branch}:master");
```

Comment:
should be `{$branch}` instead of `${branch}`

Reply:
fixed